security_config.yaml

```
@@ -1,109 +1,90 @@
     enabled: true
     requests_per_minute: 60
     burst: 100
-  
   cors:
     allowed_origins:
-      - "http://localhost:3000"  # Future UI
-      - "http://localhost:8080"
-    allowed_methods: ["GET", "POST", "PUT", "DELETE"]
-
-# Protocol Execution Security
+    - http://localhost:3000
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)